### PR TITLE
feat: add item lang in meilisearch index

### PR DESCRIPTION
**Pull request by pyphilia**
Use item lang in meilisearch.

- [x] write tests

ref #800

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
     "@fastify/type-provider-typebox": "4.1.0",
     "@fastify/websocket": "7.2.0",
     "@graasp/etherpad-api": "2.1.1",
-    "@graasp/sdk": "4.32.1",
+    "@graasp/sdk": "github:graasp/graasp-sdk#index-item-lang",
```
Comment (ReidyT) [inline package.json:66]: reminder ^^

**Review by ReidyT — APPROVED** [review]
LGTM 🦦

File: src/services/item/repository.ts [hunk 2]
```diff
@@ -141,7 +141,7 @@ export class ItemRepository extends MutableRepository<Item, UpdateItemBody> {
         ...settings,
       },
       // set lang from user lang
-      lang: lang ?? creator?.lang ?? 'en',
+      lang: lang ?? parent?.lang ?? creator?.lang ?? 'en',
```
Comment (spaenleh) [inline src/services/item/repository.ts:144]: Should `en` be defined as the default language ? is it used in multiple places ? Maybe having it as a local constant in the file could give it more visibility ? Maybe it is fine to keep it like this.

**Review by spaenleh — APPROVED** [review]
⚡

**Pull request merged**